plugin/manager_windows.go

```diff
@@ -36,3 +38,20 @@ func isExecutableFile(filePath string) (bool, error) {
 	}
 	return strings.EqualFold(filepath.Ext(filepath.Base(filePath)), ".exe"), nil
 }
+
+// parsePluginName checks if fileName is a valid plugin file name
+// and gets plugin name from it based on spec: https://github.com/notaryproject/specifications/blob/main/specs/plugin-extensibility.md#installation
+func parsePluginName(fileName string) (string, error) {
+	fname := file.TrimFileExtension(fileName)
+	pluginName, found := strings.CutPrefix(fname, proto.Prefix)
+	if !found || pluginName == "" {
```

Comment:
Should check filename has `.exe` suffix (case insensitive) in the method. Otherwise, line 250 of `manager.go` may have incorrect logic state on `notation-foo.bar` on Windows.